test/t_second_doc

```diff
@@ -23,7 +23,7 @@ EOF
 
 set -x
 echo -n > $T.out
-test_run $REQFLOW stat -s -c $T.req
-test_run $REQFLOW trac -c $T.req
+test_run ! $REQFLOW stat -s -c $T.req
```

Comment:
modification not needed after the modification of test_run.

Reply:
As I have written above in the comment about 'test_run', I think it is still needed.
As the 'reqflow' command is expected to return a non null exit status and this information is part of the test.